python/tk_multi_workfiles/file_finder.py

```diff
@@ -956,7 +982,6 @@ def _task_process_publish_items(self, sg_publishes, environment, name_map, **kwa
     def _task_find_work_files(self, environment, **kwargs):
         """
         """
-        #time.sleep(5)
```

Comment:
You have found my dirty secret. This is probably from back when I introduced deferred loading for tabs, i.e, not being loaded from disk until in foreground. ;)